endpoints/discovery_generator.py

```diff
@@ -508,7 +508,7 @@ def __params_descriptor(self, message_type, request_kind, path, method_id,
 
     return params
 
-  def __params_order_descriptor(self, message_type, path):
+  def __params_order_descriptor(self, message_type, path, is_params_class=False):
```

Comment:
nit: maybe update the docstring?

I'm actually not sure what the purpose of this parameter is.

Reply:
Done